### DISCUSSION TITLE
jwt:group in policies

### DISCUSSION THREAD

**Discussion by bpedersen2** [minio/minio#13604] [Q&A] [answered]
Looking at the docs and the test it should be possible to use jwt:groups in policies.

What I am trying to do: create a policy where all group members can access buckets with a path prefix named like one of  their groups. But as groups is a comma-separated list of groups I am having trouble as using it ..

{
        "Version":
        "2012-10-17",
        "Statement": [
            {
                "Effect": "Allow",
                "Principal": {"AWS": "*"},
                "Action": ["s3:GetBucketLocation", "s3:ListBucket"],
                "Resource": "arn:aws:s3:::my-bucket",
            },
            {
                "Effect": "Allow",
                "Principal": {"AWS": "*"},
                "Action": "s3:GetObject",
                "Resource": "arn:aws:s3:::my-bucket/*",
              "Condition": {"StringLike" : <condition> }
 
            },
        ],
    }

 condition :
        {"somegroup: "${jwt:groups}/*" } does not work
 condition 
        {"${jwt:groups"}: "somegroup/*" is not accepted

While I  have a workaround by creating a new IAM policy for each group  I am interessted ot know if this was ever supposed to work? Ideally a policy with a bucket named like the group could get created.

**Comment by harshavardhana** [comment, marked as answer]
You can use array values @bpedersen2  in policies - it is not supported yet please follow things here https://github.com/minio/minio/issues/11303
